ci-operator/step-registry/hypershift/kubevirt/install/disconnected/hypershift-kubevirt-install-disconnected-commands.sh

```diff
@@ -0,0 +1,267 @@
+    oc get clusterversion version -o jsonpath='{.status.desired.version}' | awk -F "." '{print $1"."$2}'
+}
+
+CNV_PRERELEASE_VERSION=${CNV_PRERELEASE_VERSION:-$(ocp_version)}
+CNV_PRERELEASE_CATALOG_IMAGE=${CNV_PRERELEASE_CATALOG_IMAGE:-quay.io/openshift-cnv/nightly-catalog:${CNV_PRERELEASE_VERSION}}
+
+if [ -z "${CNV_PRERELEASE_VERSION}" ]
+then
+  CNV_RELEASE_CHANNEL=stable
+  CNV_SUBSCRIPTION_SOURCE=redhat-operators
+else
+  CNV_RELEASE_CHANNEL=nightly-${CNV_PRERELEASE_VERSION}
+  CNV_SUBSCRIPTION_SOURCE=nightly-catalog
+fi
+
+# The kubevirt tests require wildcard routes to be allowed
+oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
+
+# Make the masters schedulable so we have more capacity to run VMs
+oc patch scheduler cluster --type=json -p '[{ "op": "replace", "path": "/spec/mastersSchedulable", "value": true }]'
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+scp "${SSHOPTS[@]}" "/etc/cnv-nightly-pull-credentials/openshift_cnv_pullsecret" "root@${IP}:/home/openshift_cnv_pullsecret"
+echo "$CNV_PRERELEASE_CATALOG_IMAGE" > /tmp/cnv-prerelease-catalog-image
+scp "${SSHOPTS[@]}" "/tmp/cnv-prerelease-catalog-image" "root@${IP}:/home/cnv-prerelease-catalog-image"
+echo "$CNV_PRERELEASE_VERSION" > /tmp/cnv-prerelease-version
+scp "${SSHOPTS[@]}" "/tmp/cnv-prerelease-version" "root@${IP}:/home/cnv-prerelease-version"
+
+
+# shellcheck disable=SC2087
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+set -xeo pipefail
+
+
+echo "1. Get mirror registry"
+mirror_registry=\$(oc get imagecontentsourcepolicy -o json | jq -r '.items[].spec.repositoryDigestMirrors[0].mirrors[0]')
+mirror_registry=\${mirror_registry%%/*}
+if [[ \$mirror_registry == "" ]] ; then
+    echo "Warning: Can not find the mirror registry, abort !!!"
+    exit 1
+fi
+echo "mirror registry is \${mirror_registry}"
+
+echo "3: Set registry credentials"
+yum install -y skopeo
+oc -n openshift-config extract secret/pull-secret --to="/tmp" --confirm
+set +x
+mirror_token=\$(cat "/tmp/.dockerconfigjson" | jq -r --arg var1 "\${mirror_registry}" '.auths[\$var1]["auth"]'|base64 -d)
+skopeo login "\${mirror_registry}" -u "\${mirror_token%:*}" -p "\${mirror_token#*:}"
+QUAY_USERNAME=openshift-cnv+openshift_ci
+QUAY_PASSWORD=\$(cat /home/openshift_cnv_pullsecret)
+skopeo login -u "\$QUAY_USERNAME" -p "\$QUAY_PASSWORD" quay.io/openshift-cnv
+set -x
+
+CNV_PRERELEASE_CATALOG_IMAGE=\$(cat /home/cnv-prerelease-catalog-image)
+CNV_PRERELEASE_VERSION=\$(cat /home/cnv-prerelease-version)
+# TODO: handle stable when needed
+#CNV_CHANNEL="stable"
+CNV_CHANNEL="nightly-\${CNV_PRERELEASE_VERSION}"
+
+echo "4: skopeo copy docker://\${CNV_PRERELEASE_CATALOG_IMAGE} oci:///home/cnv-local-catalog --remove-signatures"
+skopeo copy "docker://\${CNV_PRERELEASE_CATALOG_IMAGE}" "oci:///home/cnv-local-catalog" --remove-signatures
+
+echo "5. extract oc-mirror from image oc-mirror:v4.13.9"
```

Comment:
Because step: hypershift-mce-install-disconnected has already extracted the oc-mirror cli, we can add: 
```
if [[ ! -f /home/oc-mirror ]]; then
 #xxxx        
fi
```
https://github.com/openshift/release/blob/34ffb1819242e23b2c733e902414dc1dc95523d5/ci-operator/step-registry/hypershift/mce/install/disconnected/hypershift-mce-install-disconnected-commands.sh#L51